src/app/store/machine/slice.ts

```diff
@@ -638,19 +638,6 @@ const machineSlice = createSlice({
     createLogicalVolumeError: statusHandlers.createLogicalVolume.error,
     createLogicalVolumeStart: statusHandlers.createLogicalVolume.start,
     createLogicalVolumeSuccess: statusHandlers.createLogicalVolume.success,
-    createNotify: (state: MachineState, action) => {
```

Comment:
We no longer need to store new machines. Fetching machines will be handled through the list and get endpoints.